tfx/components/example_gen/driver.py

```diff
@@ -53,9 +53,10 @@ def update_output_artifact(
     exec_properties: execution properties passed to the example gen.
     output_artifact: the example artifact to be output.
   """
-  output_artifact.custom_properties[
-      utils.FINGERPRINT_PROPERTY_NAME].string_value = str(
-          exec_properties[utils.FINGERPRINT_PROPERTY_NAME])
+  if exec_properties[utils.FINGERPRINT_PROPERTY_NAME] is not None:
```

Comment:
Should we exec_properties.get(utils.FINGERPRINT_PROPERTY_NAME) to guard against the key not in this exec_properties? Or that guaranteed by an API contract?

@1025KB

Reply:
For official drivers, fingerprint will also be set (but we might set it to None when fingerprint is not available)